README.md

```diff
@@ -124,29 +126,26 @@ The easiest way to tweak react-colorful is to create another stylesheet to overr
 As you probably noticed the color picker itself doesn't include an input field, but don't worry if you need one. **react-colorful** is a modular library that allows you to build any picker you need. Since `v2.1` we provide an additional component that works perfectly in pair with our color picker.
 
 <details>
-  <summary>How to use <code>HexInput</code></summary><br />
+  <summary>How to use <code>HexColorInput</code></summary><br />
 
-```diff
-import ColorPicker from "react-colorful";
-+import HexInput from "react-colorful/HexInput";
+```js
+import { HexColorPicker, HexColorInput } from "react-colorful";
 import "react-colorful/dist/index.css";
 
 const YourComponent = () => {
   const [color, setColor] = useState("#aabbcc");
   return (
     <div>
-      <ColorPicker color={color} onChange={setColor} />
-+     <HexInput color={color} onChange={setColor} />
+      <HexColorPicker color={color} onChange={setColor} />
+      <HexColorInput color={color} onChange={setColor} />
     </div>
   );
 };
 ```
 
 [Live demo →](https://codesandbox.io/s/react-colorful-hex-input-demo-0k2fx)
 
-`HexInput` doesn't have any default styles, but accepts all properties that a regular `input` tag does (such as `className`, `placeholder` and `autoFocus`). That means you can place and modify this component as you like. Also, that allows you to combine the color picker and input in different ways.
-
-By the way, `HexInput` is also minimalist-friendly — only 400 bytes gzipped.
+`HexColorInput` doesn't have any default styles, but accepts all properties that a regular `input` tag does (such as `className`, `placeholder` and `autoFocus`). That means you can place and modify this component as you like. Also, that allows you to combine the color picker and input in different ways.
```

Comment:
I'm thinking the docs probably shouldn't be using contractions (I'm guilty of using them too) as it's less professional I guess? Totally a nitpick, but I'm going to run through quickly and fix that.